pygments/lexers/matlab.py

```diff
@@ -101,8 +101,8 @@ class MatlabLexer(RegexLexer):
             # is recognized if it is either surrounded by spaces or by no
             # spaces on both sides; only the former case matters for us.  (This
             # allows distinguishing `cd ./foo` from `cd ./ foo`.)
-            (r'(?:^|(?<=;))\s*\w+\s+(?!=|\(|(%s)\s+)' % _operators, Name,
-             'commandargs'),
+            (r'(?:^|(?<=;))(\s*)(\w+)(?=\s)(?!\s+(=|\(|(%s)\s+))' % _operators,
+             bygroups(Text, Name), 'commandargs'),
```

Comment:
The whitespace that's now in the lookahead could also be in its own Text group with bygroups().

Reply:
I can just let a later match handle that?

Reply:
Yes, sure. I just don't like to have lookaheads or lookbehinds when normal matching does the same job; they complicate the regex needlessly.

Reply:
```patch
-            (r'(?:^|(?<=;))(\s*)(\w+)(?=\s)(?!\s+(=|\(|(%s)\s+))' % _operators,
-             bygroups(Text, Name), 'commandargs'),
+            (r'(?:^|(?<=;))(\s*)(\w+)(\s+)(?!=|\(|(%s)\s+)' % _operators,
+             bygroups(Text, Name, Text), 'commandargs'),
```
works for you?

Reply:
That's what I meant, yes!

Reply:
pushed